shared/actions/config/index.js

```diff
@@ -112,13 +130,19 @@ const dispatchSetupEngineListeners = () => {
 }
 
 let _firstTimeConnecting = true
-const startHandshake = () => {
+const startHandshake = (state: TypedState) => {
```

Comment:
looks like we don't use firstTimeConnecting anywhere. And I think the version replaces it anyway (version=1 means first time)